docker/api/network.py

```diff
@@ -38,7 +38,7 @@ def networks(self, names=None, ids=None):
     @minimum_version('1.21')
     def create_network(self, name, driver=None, options=None, ipam=None,
                        check_duplicate=None, internal=False, labels=None,
-                       enable_ipv6=False):
+                       enable_ipv6=False, attachable=False):
```

Comment:
Maybe we should have `attachable=None` here, and only accept either value (True/False) if API version >= 1.24?